src/client/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenForClientParameters.cs

```diff
@@ -20,13 +20,18 @@ internal class AcquireTokenForClientParameters : IAcquireTokenParameters
         /// </summary>
         public bool AutoDetectRegion { get; set; }
 
+        /// <summary>
+        /// </summary>
+        public string RegionToUse { get; set; }
```

Comment:
Even though this is an internal class, I believe the comments made on what the data is warrents that the summary field is specified.

Reply:
Added